lib/pages/company_site/eti/popups/price_popup.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module CompanySite
+  module ETI
+    class Table
+      class PricePopup < self
+        div(:popup, css: '.popup-price')
+        radio_button_group(:price_type, css: '.js-select-type-price')
+        text_area(:price_value, css: '.pv-wrapper:not([style*="display: none"]) [name="price"]')
+        text_area(:price_max_value, css: '.pv-wrapper:not([style*="display: none"]) [name="price_max"]')
+        text_area(:discount_price_value, css: '.pv-wrapper:not([style*="display: none"]) [name="discount_price"]')
+        text_area(:discount_percent, css: '.pv-wrapper:not([style*="display: none"]) [name="product_discount_percent"]')
+        select_list(:measure, css: '.pv-wrapper:not([style*="display: none"]) #product_measure_unit_id')
+        select_list(:currency, css: '.pv-wrapper:not([style*="display: none"]) #currency')
+        text_area(:expires_at, css: '.pv-wrapper:not([style*="display: none"]) .js-discount-expires-at')
+
+        button(:save, xpath: '//*[@class="ui-button-text" and text()="Сохранить"]/..')
+        button(:cancel, xpath: '//*[@class="ui-button-text" and text()="Отменить"]/..')
+
+        def exact_price(options = {})
+          select_price_type('exact')
+          self.price_value = options[:price] if options[:price]
+          self.currency = options[:currency] if options[:currency]
+          self.measure = options[:measure] if options[:measure]
+        end
+
+        def range_price(options = {})
+          select_price_type('range')
+          self.price_value = options[:price] if options[:price]
+          self.price_max_value = options[:price_max] if options[:price_max]
+          self.currency = options[:currency] if options[:currency]
+          self.measure = options[:measure] if options[:measure]
+        end
+
+        def discount_price(options = {})
```

Comment:
Metrics/CyclomaticComplexity: Cyclomatic complexity for discount_price is too high. [7/6]